keras_cv/models/segmentation/deeplab_v3_plus/deeplab_v3_plus_test.py

```diff
@@ -74,6 +74,18 @@ def test_weights_change(self):
             self.assertNotAllEqual(w1, w2)
             self.assertFalse(ops.any(ops.isnan(w2)))
 
+    @pytest.mark.large
+    def test_with_model_preset_forward_pass(self):
+        model = DeepLabV3Plus.from_preset(
+            "deeplab_v3_plus_resnet50_pascalvoc",
+            num_classes=21,
+            input_shape=[512, 512, 3],
+        )
+        image = np.ones((1, 512, 512, 3))
+        output = model(image)
+        expected_output = np.zeros((1, 512, 512, 3))
```

Comment:
Channels shape here should be 21 to match `num_classes` -- this test failed on GCB again. Once you make the change could you run the test locally to verify that it passes?

Reply:
Thanks Ian, I just updated the test. Since my local setup is throwing random errors with loading the preset I haven't been able to run the test. Hopefully its all fixed now.